src/vos/vos_pool.c

```diff
@@ -70,23 +70,6 @@ vos_pool_pop2df(PMEMobjpool *pop)
 	return D_RW(pool_df);
 }
 
-static int
-umem_get_type(void)
-{
-	/* NB: BYPASS_PM and BYPASS_PM_SNAP can't coexist */
-	if (daos_io_bypass & IOBP_PM) {
-		D_PRINT("Running in DRAM mode, all data are volatile.\n");
-		return UMEM_CLASS_VMEM;
-
-	} else if (daos_io_bypass & IOBP_PM_SNAP) {
-		D_PRINT("Ignore PMDK snapshot, data can be lost on failure.\n");
-		return UMEM_CLASS_PMEM_NO_SNAP;
```

Comment:
the pmem_no_snap_ops in src/common/mem.c will be dead code and needs be removed as well.

Reply:
Good catch